src/main/kotlin/net/lumalyte/lg/infrastructure/persistence/migrations/SQLiteMigrations.kt

```
@@ -1383,4 +1413,112 @@ class SQLiteMigrations(private val plugin: JavaPlugin, private val connection: C
+                "UPDATE guilds SET ally_home_allowed_guilds = ? WHERE id = ?"
+            )
+            for (gid in guildIds) {
+                val allies = mutableSetOf<String>()
+                connection.prepareStatement(
+                    "SELECT guild_a, guild_b FROM relations WHERE type = 'ALLY' AND status = 'ACTIVE' AND (guild_a = ? OR guild_b = ?)"
+                ).use { ps ->
+                    ps.setString(1, gid); ps.setString(2, gid)
+                    val r2 = ps.executeQuery()
+                    while (r2.next()) {
+                        val a = r2.getString("guild_a"); val b = r2.getString("guild_b")
+                        if (a != gid) allies.add(a); if (b != gid) allies.add(b)
```

detekt (1.23.8)

[warning] This else block is empty and can be removed.

(detekt.empty-blocks.EmptyElseBlock)